packages/frontend-2/components/workspace/discoverableWorkspaces/Modal.vue

```diff
@@ -64,6 +65,10 @@ const dialogButtons = computed((): LayoutDialogButton[] => {
   ]
 })
 
+const onWorkspaceDismissed = (workspaceId: string) => {
+  localWorkspaces.value = localWorkspaces.value.filter((w) => w.id !== workspaceId)
```

Comment:
@andrewwallacespeckle Why did we have do this localWorkspaces again? Is it because we want certain workspace to still show in the UI, even though they are actually joined?